### DISCUSSION TITLE
Implementing Alt-Key Mouse Hover Selection for Efficient Node Navigation

### DISCUSSION THREAD

**Discussion by guoranjun2** [freeplane/freeplane#1663] [Ideas]
Are you tired of constantly clicking the mouse from one node to another? Perhaps we should offer a quick option to avoid this tedious operation when we need to search or sift through information intensively. My idea is to utilize the Alt key. Could we make this key a bit more useful? For instance, when we press the Alt key and move the mouse, the node where the mouse currently resides gets selected. This is much faster than continuous clicking. I've applied this approach on Swing's JTree, and I find it very comfortable. What do you think of this idea?

**Comment by euu2021** [comment]
Are you aware of `Preferences…->Behaviour->Selection method->Selection method->Direct`? 
I only use Direct and I like it.

**Reply by guoranjun2** [reply]
Thank you, I admit I missed it. combining both two way is also a flexible solution. and it occur to me just now, maybe we can add an option for automatic node unfolding?

**Reply by guoranjun2** [reply]
It can be done by script, I will try it later.

**Comment by euu2021** [comment]
>  maybe we can add an option for automatic node unfolding?

Take a look at `Navigate->Auto-expand selected nodes`. This feature was added some weeks ago.
